# Start wildfire prediction

float64 horizon